lib/ansible/modules/windows/win_iis_webbinding.py

```diff
@@ -112,20 +84,76 @@
   win_iis_webbinding:
     name: Default Web Site
     protocol: https
+    port: 443
+    certificate_hash: B0D0FA8408FC67B230338FCA584D03792DA73F4C
     state: present
 
-- name: Add a HTTPS binding and select certificate to use
+- name: Add a HTTPS binding with host header and SNI enabled
   win_iis_webbinding:
     name: Default Web Site
     protocol: https
-    certificate_hash: B0D0FA8408FC67B230338FCA584D03792DA73F4C
+    port: 443
+    host_header: test.com
+    ssl_flags: 1
```

Comment:
I wonder if we should still document this considering there is a bug with urllib3 that breaks pywinrm when SNI is enabled.

Reply:
So leave in the ability to set sni, but omit it from docs? Hmm, tough call. I actually wasn't expecting this to pass the Shippable testing until that was fixed so figured that this wouldn't come up.

It seems to only be present when people reference their servers by IP rather than host name. I'm not sure how common that is. What about adding a warning to the module description about the bug? I almost wonder if it should even be merged before that is fixed. Might be worth bringing up in the next WWG.

Reply:
Ah yes totally forgot it's only when using IP addresses so let's leave it in.

Reply:
Lol, yah. Ok cool